src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = { ...state };
+  const result = [];
+
+  for (const index of actions) {
+    switch (index.type) {
```

Comment:
```suggestion
  for (const object of actions) {
    switch (index.type) {
```
use more semantic names for variables